tsdb/engine/tsm1/iterator.gen.go.tmpl

```diff
@@ -108,7 +108,7 @@ type {{.name}}Iterator struct {
 	aux   []cursorAt
 	conds struct {
 		names []string
-		curs  []*bufCursor
+		curs  []cursorAt
```

Comment:
This is so conditions can include tag values too. If a condition only has tags, they get filtered out. If a condition has a field and a tag, then we need a fake cursor for the tag to compare it with the field.